dev/WindowsAppRuntime_BootstrapDLL/MddBootstrap.cpp

```diff
@@ -500,7 +500,7 @@ void FindDDLMViaEnumeration(
     // We need to look for DDLM packages in the package family for release <major>.<minor> and <versiontag>
     // But we have no single (simple) enumeration to match that so our logic's more involved compared
     // to FindDDLMViaAppExtension():
-    // 1. Look for Framework packages with Name="microsoft.winappruntime.ddlm.<minorversion>.*[-shorttag]"
+    // 1. Look for Framework packages with Name="microsoft.winappruntime.ddlm.<minorversion>*[-shorttag]"
```

Comment:
Shouldn't this be Major?

Example: `Microsoft.WinAppRuntime.DDLM.0.319.455.0-x8_0.319.455.0_x86__8wekyb3d8bbwe`
Major: 0
Minor: 319
Build: 455
Revision: 0

```suggestion
    // 1. Look for Framework packages with Name="microsoft.winappruntime.ddlm.<majorversion>*[-shorttag]"
```

Reply:
Nope. ByDesign. DDLM naming's weird. See other comment